Makefile

```diff
@@ -6,14 +6,14 @@ BUILD_DATE := $(shell date -u '+%Y-%m-%d_%H:%M:%S')
 BUILD_DIR = bin
 
 build-all:
-	GOOS=linux GOARCH=amd64 make build
-	GOOS=linux GOARCH=arm make build
+	make build-arm
+	make build-amd64
 
-build-prod:
-	GOOS=linux GOARCH=arm RELEASE=prod make build
```

Comment:
RELEASE feels like it should be more version - like GitHub tag to allow us to understand what it was built against :)

Reply:
Agree, but we still need a way to set the flavour we want to run here ("cloud hosted" == cloud or "home network self-hosted" == prod). But changing RELEASE to be the tag version sounds good to me :)